README.md

```diff
@@ -20,6 +20,11 @@ ab = Ab::Tests.new(configuration, identifier)
 Ab::Tests.before_picking_variant { |test| puts "picking variant for #{test}" }
 Ab::Tests.after_picking_variant { |test, variant| puts "#{variant_name}" }
 
+# by default messages are logged to null logger. Can be changed by setting Your own logger:
```

Comment:
http://english.stackexchange.com/questions/30185/you-versus-you-as-polite-form-of-writing apparently I mistaken it for a "polite" form.